### PR TITLE
Adds titanium-page-menu

### PR DESCRIPTION
**Pull request by unfairSurprisery**
It's the bubbly in-page side menu thing A-a-ron is in love with.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,12 @@
   "version": "0.0.2",
   "license": "MIT",
   "devDependencies": {
+    "core-js": "^3.1.3",
```
Comment (aarondrabeck) [inline package.json]: Whats this?

Comment (unfairSurprisery) [reply package.json]: Storybook 5.1.1 updates it's Babel dependency without updating it's core-js dep. The result is a wall of core-js errors. They're aware of it and supposedly they're going to fix it, but this is the workaround in the meantime.

Comment (unfairSurprisery) [reply package.json]: Aaaaand, they just fixed it. I'm gonna get rid of this.

**Comment by aarondrabeck** [comment]
Lets keep this branch open, but add this component to PCMS directly.  That way we can tweak it and get it working perfectly in all browsers.  Then at the end of the sprint, we can publish it in titanium elements.

**Comment by unfairSurprisery** [comment]
> Lets keep this branch open, but add this component to PCMS directly. That way we can tweak it and get it working perfectly in all browsers. Then at the end of the sprint, we can publish it in titanium elements.

Good Idea

**Pull request merged**